File: Bindings~/python/altunityrunner/commands/AltUnityTesterCommands/remove_notification_listener.py
from altunityrunner.commands.Notifications.notification_type import NotificationType
from altunityrunner.commands.base_command import BaseCommand
from altunityrunner.exceptions import InvalidParameterTypeException


class RemoveNotificationListener(BaseCommand):

    def __init__(self, connection, notification_type):
        super().__init__(connection, "deactivateNotification")

        if notification_type not in NotificationType:
            raise InvalidParameterTypeException(
                parameter_name='notification_type',
                expected_types=[NotificationType],
                received_type=type(notification_type)
            )
        self.notification_type = notification_type

    @property
    def _parameters(self):
        parameters = super()._parameters
        parameters.update(**{
            "notificationType": str(int(self.notification_type)),
        })

        return parameters

    def execute(self):
        self.connection.remove_notification_listener(self.notification_type)
        data = self.send()
        self.validate_response("Ok", data)

        return data

File: Assets/AltUnityTester/AltUnityDriver/Commands/AltUnityCommands/RemoveNotificationListener.cs
using System;
using Altom.AltUnityDriver.Notifications;

namespace Altom.AltUnityDriver.Commands
{
    public class RemoveNotificationListener : AltBaseCommand
    {
        private readonly DeactivateNotification cmdParams;

        public RemoveNotificationListener(IDriverCommunication commHandler, NotificationType notificationType) : base(commHandler)
        {
            this.cmdParams = new DeactivateNotification(notificationType);
        }
        public void Execute()
        {
            this.CommHandler.RemoveNotificationListener(cmdParams.NotificationType);
            this.CommHandler.Send(this.cmdParams);
            var data = this.CommHandler.Recvall<string>(this.cmdParams);
            ValidateResponse("Ok", data);
        }
    }
}
